tools/build-android.sh

```diff
@@ -4,6 +4,8 @@ set -e
 
 #Set Script Name variable
 SCRIPT=$(basename "${BASH_SOURCE[0]}")
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+REALM_CORE_DIR=$(dirname "${SCRIPT_DIR}")
```

Comment:
I think `REALM_CORE_DIR=$(builtin cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)` is all that's needed for this?